pkg/cli/cert/cert.go

```diff
@@ -124,7 +128,11 @@ func check(app *cli.Context, cfg *cmds.Server) error {
 		fmt.Fprintf(w, "-----------\t-------\t------\t-------")
 		for _, files := range fileMap {
 			for _, file := range files {
-				certs, _ := certutil.CertsFromFile(file)
+				certs, err := certutil.CertsFromFile(file)
+				if err != nil {
+					logrus.Debugf(err.Error())
+					continue
+				}
```

Comment:
nit: this is in the table format output, maybe print the file as `status = "NOT FOUND"` with empty expiration, or something like that?